zipline/data/_array_utilities.pyx

```diff
@@ -0,0 +1,52 @@
+import numpy as np
+cimport numpy as np
+cimport cython
+
+from bcolz.carray_ext cimport carray
+
+cdef inline int int_min(int a, int b): return a if a <= b else b
+
+@cython.cdivision(True)
+def minute_value(np.ndarray[np.long_t, ndim=1] arr, Py_ssize_t pos):
+    cdef short q, r
+
+    q = cython.cdiv(pos, 390)
```

Comment:
With this being a function not associated with the `BcolzMinuteBarClass`, it may make sense to pass the `minutes_per_day` as a parameter, instead of hardcoding to 390.